glfw.go

```diff
@@ -96,19 +113,28 @@ func (m *windowManager) glfwCursorPosCallback(window *glfw.Window, x, y float64)
 	m.sendPointerEvent(window, m.pointerPhase, x, y)
 }
 
-func (m *windowManager) glfwMouseButtonCallback(window *glfw.Window, key glfw.MouseButton, action glfw.Action, mods glfw.ModifierKey) {
-	if key == glfw.MouseButton1 {
-		x, y := window.GetCursorPos()
+func (m *windowManager) sendButtonEvent(window *glfw.Window, action glfw.Action, buttons embedder.PointerButtonMouse) {
```

Comment:
`sendButtonEvent` doesn't actually send a event. Perhaps this can be inlined into `glfwMousebuttonCallback`

Reply:
I created `sendButtonEvent` to avoid code replication. If not used we would have the following block everytime :
```
if action == glfw.Press {
		m.sendButtonPointerEvent(window, embedder.PointerPhaseDown, buttons)
		m.pointerPhase = embedder.PointerPhaseMove
	}

	if action == glfw.Release {
		m.sendButtonPointerEvent(window, embedder.PointerPhaseUp, buttons)
		m.pointerPhase = embedder.PointerPhaseHover
	}
```

Reply:
Ah yes, I was thinking about placing this below the switch, and having the switch set some variables. Separate function can work as well when the name is more descriptive.